acp-model/src/commonMain/kotlin/com/agentclientprotocol/model/Methods.kt

```diff
@@ -43,16 +55,16 @@ public open class AcpMethod(public val methodName: MethodName) {
     public object AgentMethods {
         // Agent-side operations (methods that agents can call on clients)
         public object Initialize : AcpRequestResponseMethod<InitializeRequest, InitializeResponse>("initialize", InitializeRequest.serializer(), InitializeResponse.serializer())
-        public object Authenticate : AcpRequestResponseMethod<AuthenticateRequest, AuthenticateResponse>("authenticate", AuthenticateRequest.serializer(), AuthenticateResponse.serializer())
+        public object Authenticate : AcpRequestResponseNullableMethod<AuthenticateRequest, AuthenticateResponse>("authenticate", AuthenticateRequest.serializer(), AuthenticateResponse.serializer())
```

Comment:
why not just do `AcpRequestResponseMethod<AuthenticateRequest, AuthenticateResponse?>`